src/rmplan/rmplan.ts

```diff
@@ -482,10 +494,47 @@ program
     '--workspace <id>',
     'ID for the task, used for workspace naming and tracking. If provided, a new workspace will be created.'
   )
+  .option('--auto-workspace', 'Automatically select an available workspace or create a new one')
+  .option(
+    '--prefer-new-workspace',
+    'When using --auto-workspace, prefer creating a new workspace over using existing ones'
+  )
```

Comment:
Change this to just `--new-workspace` and have it use this behavior:

- When `auto-workspace` is passed, then always create a new workspace
- When `workspace` is passed, this allows a new workspace to be created. If `new-workspace` is not present then refuse to create a new workspace and throw an error instead.